bundles/org.openhab.binding.tplinkrouter/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tplinkrouter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="router">
+
+		<label>Router</label>
+		<description>Router device monitored and controlled by telnet connection</description>
+
+		<channel-groups>
+			<channel-group id="wifi" typeId="wifiGroupType"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="port" type="integer">
+				<default>23</default>
```

Comment:
Did you check that your XML is valid ?
I have a doubt if the "default" tag can be before "label"...